test/api/node-after.js

```diff
@@ -0,0 +1,150 @@
+"use strict";
```

Comment:
Please remove these test files. See https://github.com/tmpvar/jsdom/blob/master/Contributing.md#tests

> All tests for web platform features (as opposed to features of jsdom itself, such as the JSDOM() constructor) should be in web-platform-tests format.

The existing tests already in the submodule will suffice. If you have cases you think are not covered by those tests, please write them in the to-upstream format, per Contributing.md.